tests/e2e/support/objects/app-files/page/public.ts

```diff
@@ -72,6 +72,15 @@ export class Public {
     await this.#page.locator('body').click()
   }
 
+  async uploadInternal(args: Omit<uploadResourceArgs, 'page'> & { link: string }): Promise<void> {
+    // link is the public link url
+    const { link } = args
+    delete args.link
+    await uploadResource({ ...args, page: this.#page })
+    await this.#page.goto(link)
+    await this.#page.locator('body').click()
```

Comment:
Do we need to take these steps? I think it's unnecessary

Reply:
was just a copy-paste. I will have a look. Thanks!